### ISSUE TITLE
Call for help

### ISSUE THREAD

**Issue by Tiriel** [nodejs/outreach#4]
# Outreach needs you!
Hi everyone!

Since its inception and first discussions last year, and formalization at the last Spring Collab Summit, the Outreach Initiative has had a very long (too long?) stasis period.
As I now have some bandwidth to work on this, I'd like us to move forward and make it real, and finally have the inclusivity and diversity platform the great developers of the Node.js Community deserve.

## What is the Outreach Initiative
The Outreach community is meant to provide help, guidance, and spotlight to under-represented groups of developers in the Node.js broad community

It's scoped to the Node.js Community at first but aims to publicize the work of the Node.js developers in the IT community as a whole (meaning that it won't restrict itself to publishing content in the Node.js GitHub org or specific media)

## TODO:
These are largely inspired/copied from @bnb 's comments in [@keywordnew 's issue](https://github.com/nodejs/outreach/issues/2#issuecomment-418718995).
* Rework, reword, maybe expand, and even maybe redefine the above-stated definition and scope 
* Flesh out a proper README.md
* Create a Contributing.md file

## What we need
Anyone that's interested in giving more light to people from diverse backgrounds.

If you have any question or remarks, please feel free to ask them. If you want to join in, please do.
If you have some corrections to make to this issue or suggestions, I'd very much like to improve.
This is a journey for everyone, and I'd like to make it with the broadest range of people I can find.

Cheers!

**Comment by amiller-gh** [comment]
👋 Humble proposal – I'd like to create some concrete program opportunities that the Outreach Initiative can work on to help democratize access to on-boarding, and help provide tangible resources to all communities. The aim should be help foster an established and healthy local meetup network that CommComm can help promote and grow with D&I as a primary guiding principle (as it should be in all things!) 👍

Jotted down thoughts on what this can look like here: https://docs.google.com/document/d/14xOqRPh61wX98GTqvUJVjQsRUJ2YZyK8aYMD9Un-gMI/edit?usp=sharing

Let's get a kickoff meeting on the books for this initiative reboot for next week? 🙂

**Comment by WaleedAshraf** [comment]
I'd like to discuss this very old issue. I couldn't follow up for a long time.
https://github.com/nodejs/community-committee/issues/280

**Comment by WaleedAshraf** [comment]
@keywordnew Like your idea of using Nodejs as guiding them to contribute to open-source.
I hope you can jump into our next meeting around this.
@amiller-gh reminder to add me.

**Comment by Amorelandra** [comment]
Hey, friends. This seems like something I could potentially be interested in contributing to. I'd love to drop in on the next meeting to learn more about existing plans & how I could help. Would that be possible? Thanks!

**Comment by Tiriel** [comment]
Hey everyone! Glad to have been able to finally talk about this yesterday.

As said during the CommComm meeting, I'll post a doodle to find a suitable time next week for a first meeting. Everyone interested can then post their preferences.

In the meantime, just so that's clear, **I'd like to restate the primary goals of this initiative as I see them** :
- The main goal is to empower people from marginalized communities and more broadly minorities.
- **This will come only by letting people express themselves**, by giving them a framework, a platform, and letting them **make it their own**.
- One of the first ideas is to promote and support the development of local communities and events around the world, especially in underrepresented areas.

Of course, everything is up to discussion, I'm not the leader or anything, I don't decide, that's just my vision of the initiative.

Coming back real quick with the doodle!

**Comment by amiller-gh** [comment]
@Amorelandra so excited to hear that!!! Can't wait to see you in the meetings 🙂 

@Tiriel – bump on that doodle 😉

**Comment by Tiriel** [comment]
@amiller-gh Thanks, sorry for the lagg!

@all here's the doodle, don't hesitate to propose new times and dates here if nothing matches your schedule, I'll try to do what I can!
https://doodle.com/poll/pt396rfmahs9xrks

**Comment by MaedahBatool** [comment]
🙌 Hey, @Tiriel!  

Thanks for creating it. The proposed timings look awesome. However, IMHO Mondays and Fridays should be skipped for the obvious reasons. 🙈

Anywho, let's wait and see for the majority to vote. 

Cheers!

**Comment by keywordnew** [comment]
I'm not aware of reasons Mondays and Fridays aren't good options :) I'd appreciate the context.

**Comment by MaedahBatool** [comment]
🙌 Hey, @keywordnew!

In my opinion, Monday being the start of the week is the busiest day and Friday being the last day brings a bit of relaxation. 🙈

It may vary for person to person but I am sharing my personal experience.

**Comment by keywordnew** [comment]
Thanks @MaedahBatool! 👏🏽

**Comment by Tiriel** [comment]
Hey everyone!

Following doodle, first meeting will be tomorrow, April 2 at 16:00 UTC:
https://www.timeanddate.com/worldclock/fixedtime.html?iso=20190402T1600

@amiller-gh Do you think it will be possible to host it via the Node.js Zoom account?
Otherwise I'll set up something.

Cheers!

**Comment by MaedahBatool** [comment]
Awesome! Looking forward to it.

**Comment by MaedahBatool** [comment]
🙌 Hey, folks!

Today's meeting will be hosted via the same Zoom account?

**Comment by Tiriel** [comment]
Hey!

No idea yet. If we can, it will indeed be hosted via the same account, otherwise I'll try to set up something, but it won't be broadcasted on youtube if it comes from me. :(

**Comment by MaedahBatool** [comment]
Makes sense. Let wait till the meeting time. Maybe something gets figured. 🤞

**Comment by amiller-gh** [comment]
Hey all! Sorry for the delay :) I can host it on the CommComm channel this time 👍 See y’all in ~1.5hrs!

**Comment by Tiriel** [comment]
Awesome! Thanks @amiller-gh !

**Comment by ahmadawais** [comment]
Is there a meeting tonight or are we going to discuss this in the Node.js Foundation Community Committee Meeting 2019-04-04 #461 — asking for a friend? 🤣

**Comment by amiller-gh** [comment]
We're doing the first ad-hoc meeting tonight! (~10 minutes, morning for me 🙂) After today we'll be on a regular Tuesday bi-weekly cadence at this time, with the same automagic meeting minutes issues and dedicated Zoom webinar, that other working groups have.

**Comment by MaedahBatool** [comment]
Awesome!

**Comment by amiller-gh** [comment]
Join link: https://zoom.us/j/526318636

**Comment by Tiriel** [comment]
Quick update,

I'm terribly sorry everyone, I got some awfully urgent problem to fix and can't attend my own meeting and I feel terrible. if you want to reschedule, don't hesitate to leave a note, otherwise, I'm not indispensable.

**Comment by amiller-gh** [comment]
😢 No worries @Tiriel, I hope your problem is resolved quickly! We'll still meet briefly today and livestream it.

**Comment by MaedahBatool** [comment]
No worries @Tiriel take your time. We'll just do a quick round up.

**Comment by WaleedAshraf** [comment]
Sorry, I also missed it 🤦‍♂️
See you guys in next one.

**Comment by amiller-gh** [comment]
Great chat / inaugural meeting today 🎉

Livestream video is here: https://www.youtube.com/watch?v=v1rARhO1ekI
I've PR'd the initiative into the CommComm README here: https://github.com/nodejs/community-committee/pull/462

Additional Action Items:
 - @MaedahBatool to take first pass at `nodejs/outreach` README
 - @amiller-gh to add this initiative to the `create-node-meeting-artifacts` tool

**Comment by codeekage** [comment]
So sorry for missing this! ❤️ BTW meeting notes looking good 😊
